src/dvclive/live.py

```diff
@@ -265,10 +265,14 @@ def _init_dvc(self):  # noqa: C901
             self._include_untracked.append(self.dir)
 
     def _init_dvc_file(self) -> str:
+        if isinstance(self._dvcyaml, Path):
+            self._dvcyaml = str(self._dvcyaml)
         if isinstance(self._dvcyaml, str):
             if os.path.basename(self._dvcyaml) == "dvc.yaml":
                 return self._dvcyaml
             raise InvalidDvcyamlError
```

Comment:
Can we drop this line if we are adding the exception below?

Reply:
Yep, makes sense.